aptos-move/aptos-vm/src/counters.rs

```diff
@@ -27,15 +31,20 @@ pub static BLOCK_EXECUTOR_CONCURRENCY: Lazy<IntGauge> = Lazy::new(|| {
     .unwrap()
 });
 
+const SIGNATURE_VERIFICATION_TIME_BUCKETS: [f64; 18] = [
+    0.0002, 0.0004, 0.0006, 0.0008, 0.001, 0.00125, 0.0015, 0.002, 0.0025, 0.003, 0.0035, 0.004,
+    0.005, 0.006, 0.008, 0.01, 0.015, 0.02,
+];
+
```

Comment:
If this isn't becoming a problem, I would leave it as the default. Let's check the grafana links from the e2e test after they are done.